pycaret/containers/models/time_series/time_series_clustering.py

```diff
@@ -0,0 +1,672 @@
+
+        eq_function = lambda x: type(x) is NaiveForecaster and x.sp != 1
+
+        super().__init__(
+            id="snaive",
+            name="Seasonal Naive Forecaster",
+            class_def=NaiveForecaster,
+            args=args,
+            tune_grid=tune_grid,
+            tune_distribution=tune_distributions,
+            tune_args=tune_args,
+            is_gpu_enabled=self.gpu_imported,
+            eq_function=eq_function,
+        )
+
+    @property
+    def _set_args(self) -> Dict[str, Any]:
+        args = {"sp": self.sp}
+        return args
+
+    @property
+    def _set_tune_grid(self) -> Dict[str, List[Any]]:
+        tune_grid = {
+            "strategy": ["last", "mean"],
+            "sp": [self.sp, 2 * self.sp],
+            # Removing fh for now since it can be less than sp which causes an error
+            # Will need to add checks for it later if we want to incorporate it
+            "window_length": [None],  # , len(fh)]
+        }
+        return tune_grid
+
+
+class TimeSeriesKernelKMeansContainer(TimeSeriesKMeansContainer):
+    model_type = TSModelTypes.BASELINE
+
+    def __init__(self, experiment) -> None:
+        self.logger = get_logger()
+        np.random.seed(experiment.seed)
+        self.gpu_imported = False
+
+        from sktime.forecasting.trend import PolynomialTrendForecaster  # type: ignore
+
+        # Disable container if certain features are not supported but enforced ----
+        dummy = PolynomialTrendForecaster()
+        self.active = _check_enforcements(forecaster=dummy, experiment=experiment)
+        if not self.active:
+            return
+
+        args = self._set_args
+        tune_args = self._set_tune_args
+        tune_grid = self._set_tune_grid
+        tune_distributions = self._set_tune_distributions
+        leftover_parameters_to_categorical_distributions(tune_grid, tune_distributions)
+
+        super().__init__(
+            id="polytrend",
+            name="Polynomial Trend Forecaster",
+            class_def=PolynomialTrendForecaster,
+            args=args,
+            tune_grid=tune_grid,
+            tune_distribution=tune_distributions,
+            tune_args=tune_args,
+            is_gpu_enabled=self.gpu_imported,
+        )
+
+    @property
+    def _set_tune_grid(self) -> Dict[str, List[Any]]:
+        tune_grid = {"degree": [1, 2, 3, 4, 5], "with_intercept": [True, False]}
+        return tune_grid
+
+    @property
+    def _set_tune_distributions(self) -> Dict[str, List[Any]]:
+        tune_distributions = {
+            "degree": IntUniformDistribution(lower=1, upper=10),
+            "with_intercept": CategoricalDistribution(values=[True, False]),
+        }
+        return tune_distributions
+
+
+######################################
+# CLASSICAL STATISTICAL MODELS ####
+######################################
+
+
+class TimeSeriesLloydsContainer(TimeSeriesKMeansContainer):
```

Comment:
Hi @skanjila I am assuming these are just dummy placeholders for now. Is this correct?